### ISSUE TITLE
removeSecondaryCursors should leave the cursor added last

### ISSUE THREAD

**Issue by niobium0** [microsoft/vscode#80607]
`removeSecondaryCursors` (i.e. clicking "Escape" after selecting multiple cursors), should leave the cursor which was added last. This allows fast searching for other uses in the file (`ctrl-d ctrl-d esc` as opposed to `ctrl-d ctrl-c ctrl-f ctrl-v enter esc`).

Sublime and Atom implement this by default.

**Comment by alexandrudima** [comment]
This is not the behaviour of Sublime. I have tried Cmd+D, Cmd+D, Escape, and the cursor was left on the first word.

**Comment by niobium0** [comment]
> This is not the behaviour of Sublime. I have tried Cmd+D, Cmd+D, Escape, and the cursor was left on the first word.

You are right. Apparently neither it works in Atom. Apologies for the confusion.
